media/src/components/UsersToFollow.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Text,
+  Heading,
+  Grid,
+  GridItem,
+  Avatar,
+  Button,
+  Card,
+  Flex,
+  Box,
+  Center,
+} from "@chakra-ui/react";
+import { faker } from "@faker-js/faker";
+
+const UsersToFollow = () => {
+  // Sample user data
+  const users = [
+    {
+      id: 1,
+      name: "John Doe",
+      username: "@johndoe",
+      avatar: "https://example.com/avatar1.jpg",
+    },
+    {
+      id: 2,
+      name: "Jane Smith",
+      username: "@janesmith",
+      avatar: "https://example.com/avatar2.jpg",
+    },
+    {
+      id: 3,
+      name: "Jane Smith",
+      username: "@johnDoe",
+      avatar: "https://example.com/avatar2.jpg",
+    },
+    {
+      id: 4,
+      name: "Jane Smith",
+      username: "@Michealsmith",
+      avatar: "https://example.com/avatar2.jpg",
+    },
+    // Add more user data here
+  ];
+
+  return (
+    <Card
+      borderWidth="1px"
+      borderRadius="lg"
+      p={4}
+      shadow="md"
+      bg="#3c096c"
+      width={"100%"}
+    >
+      <Heading size="md" mb={3} color={"white"}>
+        Users to Follow
+      </Heading>
+      <Grid templateRows="repeat(3, 1fr)" gap={3}>
+        {users.map((user) => (
```

Comment:
i might suggest pulling the internals here into a <UserCard /> component or the like. how many indents isn't indicative of good or bad code, but it can be something to be aware of. grid -> users.map -> gridItem -> flex -> box -> text -> {username} is a lot different than grid -> users.map -> <User {...user} />